llama_stack/providers/utils/inference/litellm_openai_mixin.py

```diff
@@ -334,6 +336,7 @@ async def openai_chat_completion(
             api_key=self.get_api_key(),
             api_base=self.api_base,
         )
+        logger.info(f"params to litellm (openai compat): {params}")
```

Comment:
avoid this, it'll not only produce a lot of output but it'll also log user data

Reply:
Good catch, thx!  I put that in there thinking I wasn't pushing this file anyway, and then forgot to take it out when we wound up moving the b64_encode_openai_embeddings_response method in to the class.  Fixed now.